h2o-algos/src/main/java/hex/glm/GLMModel.java

```diff
@@ -596,9 +612,8 @@ public final double linkInvDeriv(double x) {
         case inverse:
           double xx = (x < 0) ? Math.min(-1e-5, x) : Math.max(1e-5, x);
           return -1 / (xx * xx);
-//        case tweedie:
-//          double vp = (1. - _tweedie_link_power) / _tweedie_link_power;
-//          return (1/ _tweedie_link_power) * Math.pow(x, vp);
+        case tweedie:
+          return _link_power==0?Math.max(x, Double.MIN_NORMAL):x*_oneOLinkPower*_oneOeta;
```

Comment:
The link function depends on the link power q  as follows:

<img width="260" alt="image" src="https://user-images.githubusercontent.com/7231712/58065853-f1b57a00-7b3b-11e9-808a-cf4c20aa26f1.png">

I am just implementing this.